drivers/ovsSwitch.go

```diff
@@ -95,7 +95,7 @@ func NewOvsSwitch(bridgeName, netType, localIP string, fwdMode string,
 			datapath = "vrouter"
 		default:
 			log.Errorf("Invalid datapath mode")
-			return nil, errors.New("Invalid forwarding mode. Expects 'bridge' or 'routing'")
+			return nil, errors.New("invalid forwarding mode. Expects 'bridge' or 'routing'")
```

Comment:
This is also the convention that [our errored repo](https://github.com/contiv/errored) follows